errcheck/errcheck.go

```diff
@@ -241,19 +252,19 @@ func (c *Checker) CheckPackage(pkg *packages.Package) []UncheckedError {
 	return v.errors
 }
 
-// UpdateNonVendoredIgnore updates the Ignores to use non vendored paths
-func (c *Checker) UpdateNonVendoredIgnore() {
-	if isGoMod() {
-		ignore := make(map[string]*regexp.Regexp)
-		for pkg, re := range c.Ignore {
-			if nonVendoredPkg, ok := nonVendoredPkgPath(pkg); ok {
-				ignore[nonVendoredPkg] = re
-			} else {
-				ignore[pkg] = re
-			}
+// getNonVendoredIgnores gets the ignore expressions for non-vendored packages.
+// They are returned as a map from package path names to '.*' regular
+// expressions.
+func (c *Checker) getNonVendoredIgnores() map[string]*regexp.Regexp {
```

Comment:
This could be a standalone function that takes `[]string` since the only dependency it uses is `c.Exclusions.Packages`

Reply:
Refactored; this function was dissolved altogether.